<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>실시간 거래대금 급증 탐지 시스템</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_control">
      <property name="title">
       <string>제어 패널</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_account">
         <property name="text">
          <string>계좌번호:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="account_info">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>연결 상태:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="connection_status">
         <property name="styleSheet">
          <string>color: red; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>미연결</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_current_time">
         <property name="text">
          <string>현재시간:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="current_time">
         <property name="font">
          <font/>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>--:--:--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_button">
         <property name="text">
          <string>시작</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>중지</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_parameters">
      <property name="title">
       <string>알림 조건 설정</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="condition_combobox"/>
       </item>
       <item row="0" column="10">
        <widget class="QLabel" name="monitoring_count">
         <property name="styleSheet">
          <string>font-weight: bold; color: #2196F3;</string>
         </property>
         <property name="text">
          <string>0개</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QDoubleSpinBox" name="amount_multiplier">
         <property name="suffix">
          <string>배</string>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>20.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_min_amount">
         <property name="text">
          <string>최소 거래대금:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="lookback_candles">
         <property name="suffix">
          <string>개</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="min_amount">
         <property name="suffix">
          <string>억원</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_condition_index">
         <property name="text">
          <string>조건검색:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_lookback_candles">
         <property name="text">
          <string>비교할 이전 분봉 개수:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="body_tail_ratio">
         <property name="suffix">
          <string>배</string>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.200000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_body_tail_ratio">
         <property name="text">
          <string>몸통/윗꼬리 최소 비율:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_amount_multiplier">
         <property name="text">
          <string>거래대금 증가 배수:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QLabel" name="label_monitoring_count">
         <property name="text">
          <string>모니터링 종목:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_program_count">
         <property name="text">
          <string>프로그램 순매수 상위</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="program_count">
         <property name="suffix">
          <string>위 이내</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_alerts">
      <property name="title">
       <string>알림 목록</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="alert_table">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>시간</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>종목코드</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>종목명</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>시가</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>고가</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>저가</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>종가</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>거래대금</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>이전평균</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>배수</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_log">
      <property name="title">
       <string>로그</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextBrowser" name="log_browser">
         <property name="styleSheet">
          <string>font-family: 'Consolas', 'Courier New', monospace; font-size: 9pt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
